experiments/azure/cyclecloud/cpu/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:bookworm
+
```

Comment:
Is this accidentally added to cyclecloud?

Reply:
Yeah, and now removed from the PR.